include/Util/BasicTypes.h

```diff
@@ -219,6 +251,67 @@ class SVFFunction : public SVFValue
         return this->exitBB;
     }
 
+    bool dominate(const BasicBlock* bbKey, const BasicBlock* bbValue) const 
+    {
+        if (bbKey == bbValue)
+            return true;
+        
+        // An unreachable node is dominated by anything.
+        if (isUnreachable(bbValue)){
+            return true;
+        }
+
+        // And dominates nothing.
+        if (isUnreachable(bbKey))
+        {
+            return false;
+        }
+        
+        const Map<const BasicBlock*,Set<const BasicBlock*>>& dtBBsMap = getDomTreeMap();
+        Map<const BasicBlock*,Set<const BasicBlock*>>::const_iterator mapIter = dtBBsMap.find(bbKey);
+        if (mapIter != dtBBsMap.end())
+        {
+            const Set<const BasicBlock*> & dtBBs = mapIter->second;
+            Set<const BasicBlock*>::const_iterator setIter = dtBBs.find(bbValue);
+            if (setIter != dtBBs.end())
```

Comment:
Since `setIter` is only used once, simply do `dtBBs.find(bbValue) != dtBBs.end()` -- less noise.